hikari/errors.py

```diff
@@ -308,7 +308,27 @@ def __str__(self) -> str:
 
         value = super().__str__()
         if self.errors:
-            value += "\n" + data_binding.dump_json(self.errors, indent=2)
+            value += "\n\n"
+
+            def dump(obj: data_binding.JSONObject, obj_string: str = "") -> str:
+                string = ""
+                for k, v in obj.items():
+                    if isinstance(v, typing.Sequence):
+                        errors_string = obj_string + ":"
+
+                        for item in v:
+                            errors_string += f"\n - {item['message']}"
```

Comment:
Might be worth it not assuming the error will always be in this format and instead falling back to a more basic handling

Reply:
Added a fallback for when a KeyError occurs to just dump the errors JSON (the previous behaviour). Personally I think that the errors structure is well defined enough in the docs for this not be an issue (can also be seen in the wild), but better safe than sorry ig.

Also, TIL that `str` failing when raising an exception will just add `str failed` as the str of the exception